Source/Fuse.Nodes/Visual.ZOrder.uno

```diff
@@ -14,9 +14,14 @@ namespace Fuse
 		{ 
 			get
 			{
-				if (!HasVisualChildren) return null;
-
-				return FirstChild<Visual>();
```

Comment:
I see no reason to change this bit since `FirstChild` uses the new iteration. You're just inlining a copy of it here.